cloudhub-2/modules/ROOT/pages/ch2-deploy-api.adoc

```diff
@@ -0,0 +1,180 @@
+= Deploy an App with TCP using the Application Manager API
+
+To configure protocols other than HTTP or HTTPS, configure TCP inbound rules to manage traffic for your private space. You can then use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/amc-application-manager/minor/4.0/console/method/%231158/[Application Manager API] to deploy an app using TCP. 
+
+To configure firewall rules using TCP, see xref:ps-config-fw-rules.adoc#configure-firewall-rules[Configure Firewall Rules] and select TCP.
+
+== Find Available Ports for Your App
+
+The port range for TCP is from 30500 to 32500. To confirm port availability, use the https://anypoint.mulesoft.com/exchange/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/cloudhub-20-api/minor/1.0/console/method/%232327/[CloudHub 2.0 API] that returns a list of available ports. 
+
+The following examples show the API request and response when you use it to find available ports:
+
+Request:
+
+[source,json]
+---- 
+GET
+https:/anypoint.mulesoft.com/runtimefabric/api/organizations/{organizationId}/privatespaces/{privatespaceId}/ports?available=true&count=10
+----
+
+Response: 
+
+[source,json]
+----
+{
+   "ports": [
+       30500,
+       30501,
+       30502,
+       30503,
+       30504,
+       30511,
+       30512,
+       30513,
+       30514,
+       30515
+   ]
+}
+----
+
+
+The `available` parameter enables you to alternate between used and unused ports.
+The `count` parameter enables you to specify the amount of ports desired. `count` can range from 1 to 10. The end result is limited by the actual availability of ports.
```

Comment:
Should lines 42-43 be closer to the Request?

Avoid starting a sentence with lowercase.